### PR TITLE
Get tables format

### PR DESCRIPTION
**Pull request by ondrajodas**
Solving - https://keboola.atlassian.net/browse/COM-25

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/MySQL.php [hunk 1]
```diff
@@ -189,33 +191,61 @@ public function getTables(?array $tables = null): array
...
-                'type' => (isset($table['TABLE_TYPE'])) ? $table['TABLE_TYPE'] : '',
-                'rowCount' => (isset($table['TABLE_ROWS'])) ? $table['TABLE_ROWS'] : '',
-                'columns' => [],
-            ];
+            $tableFormat = new Table();
+            $tableFormat
+                ->setName($table['TABLE_NAME'])
+                ->setSchema((isset($table['TABLE_SCHEMA'])) ? $table['TABLE_SCHEMA'] : '')
+                ->setType((isset($table['TABLE_TYPE'])) ? $table['TABLE_TYPE'] : '')
+                ->setRowCount((isset($table['TABLE_ROWS'])) ? (int) $table['TABLE_ROWS'] : 0);
```
Comment (pivnicek) [inline src/Keboola/DbExtractor/Extractor/MySQL.php]: oh, please no.  there is an important difference between `0` and `unknown`! :-)

File: src/Keboola/DbExtractor/Extractor/MySQL.php [hunk 2]
```diff
@@ -189,33 +191,61 @@ public function getTables(?array $tables = null): array
...
+
             if ($table['TABLE_COMMENT']) {
```
Comment (pivnicek) [inline src/Keboola/DbExtractor/Extractor/MySQL.php]: we will probably need to add `isset` here when moving to 7.4 right?

Comment (ondrajodas) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: yes you're right

File: src/Keboola/DbExtractor/Extractor/MySQL.php (hunk 1 above)
Comment (ondrajodas) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: 🤦‍♂

**Pull request merged**